src/BlockTypes/MiniCart.php

```diff
@@ -189,6 +239,19 @@ protected function append_script_and_deps_src( $script ) {
 		);
 	}
 
+	/**
+	 * Returns the markup for render the tax label.
+	 *
+	 * @return string
+	 */
+	protected function get_include_tax_label_markup() {
+		$cart_controller     = $this->get_cart_controller();
+		$cart                = $cart_controller->get_cart_instance();
+		$cart_contents_total = $cart->get_subtotal();
+
+		return ( ! empty( $this->label_including_tax ) && 0 !== $cart_contents_total ) ? ( "<small class='wc-block-mini-cart__tax_label'>" . $this->label_including_tax . '</small>' ) : '';
```

Comment:
We should escape this label:
```suggestion
		return ( ! empty( $this->label_including_tax ) && 0 !== $cart_contents_total ) ? ( "<small class='wc-block-mini-cart__tax_label'>" . esc_html( $this->label_including_tax ) . '</small>' ) : '';
```